drivers/ddr/imx/phy/helper.c

```diff
@@ -17,7 +17,7 @@
 DECLARE_GLOBAL_DATA_PTR;
 
 #define IMEM_LEN 32768 /* byte */
-#if CONFIG_IS_ENABLED(IMX8M)
+#if IS_ENABLED(CONFIG_IMX8M)
```

Comment:
cosmetic: could you please just add explanation to commit message that `CONFIG_IS_ENABLED($option)` checks for CONFIG_SPL_$option when building SPL, this is why we need this change (TBH I  could not understand for the first 5 mins what the difference was and why it works differently)

Other than that LGTM